src/App.js

```diff
@@ -1,27 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import Screen from './components/screen';
+import ButtonsContainer from './components/buttons-container';
+import Button from './components/button';
+
+// const buttonList = [
+// 'AC', '+/-', '%', '/', '7', '8', '9', 'X', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=',
+// ];
```

Comment:
- [ ] Please remove all commented code from your codebase because it's a bad practice to commit commented-out code. That's what version control is for. If you need that code back, just fetch it from the repo :smile:

Reply:
It's done